app/views/splash_screen/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<section id="splash_screen">
+    <h1>Budget App</h1>
+    <div class="actions">
+        <%= link_to "LOG IN", new_user_session_path %>
+        <%= link_to "SIGN OUT", new_user_registration_path %>
```

Comment:
- [ ] Your button on the splash screen reads sign out, instead of sign up. Kindly fix that.

  ![image](https://github.com/Zohra-Neda/Budget-App/assets/54848363/6fee3f67-157c-4e32-b264-84e5652903d2)